### DISCUSSION TITLE
An important request about promoting the mod to another version

### DISCUSSION THREAD

**Discussion by DDOSDAFTDDOS** [Sinytra/Connector#1103] [Ideas]
Hello, I kindly ask you to add a mod for version 1.19.2 of Forge.
This version has a lot of cool mods for both forge and fabric.

**Comment by coco875** [comment]
Same answer that every other question about backport, no backport are planned. (This includes 1.19.2 and other versions below)

**Reply by DDOSDAFTDDOS** [reply]
So this mod will be on 1.19.2 in the future? you can answer simply yes or no. I'm just Russian and I don't understand Google translation.

**Reply by coco875** [reply]
By any I mean no sorry for the false hope

**Reply by marenmasak-pixel** [reply]
Hi
I'm the same as that guy from Russia, and even though it's stupid, I'll ask. You wrote that the mod won't be made for versions LOWER than 1.20.1, but will you be upgrading the mod to, say, 1.21.4?

**Reply by kakashi1464** [reply]
There won’t be an update for 1.21.4, only for 26.1

**Comment by kakashi1464** [comment]
> So this mod will be on 1.19.2 in the future? you can answer simply yes or no. I'm just Russian and I don't understand Google translation.

Бэкпороты для версий ниже 1.20.1 не запланированы (не будет коннектора для 1.19.2)
No backports below mc 1.20.1 are planned (no connector for mc 1.19.2)

**Reply by DDOSDAFTDDOS** [reply]
Very unfortunate ;<
Do you know how you can run fabric on 1.19.2 forge?
Most mods for 1.19.2 are very cool, but they are only for 1.19.2
I hope there will be more similar projects like yours.

**Reply by kakashi1464** [reply]
I am not the author of Connector. And now there is a no way to run the Fabric mod on Forge on 1.19.2

**Reply by coco875** [reply]
(You can backport the mod yourself but it will be hard or else port mod to 1.20.1)

**Reply by peaksuperior885** [reply]
could i try fork it and then backport it to older versions of mc such as 1.19.2 1.18.2 and 1.16.2?

**Reply by coco875** [reply]
the code are under the license MIT so yes

**Reply by peaksuperior885** [reply]
thanks

**Reply by peaksuperior885** [reply]
oh and sorry for late response

**Comment by peaksuperior885** [comment]
i could try again but i would need to get better at coding and get a
stronger team but i can't now
